File: app/src/main/java/com/example/atlasdevicesync/bean/Posizione.java
package com.example.atlasdevicesync.bean;

import java.util.ArrayList;

import io.realm.kotlin.types.EmbeddedRealmObject;

public class Posizione implements EmbeddedRealmObject {
    String type;
    ArrayList<Float> coordinates;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public ArrayList<Float> getCoordinates() {
        return coordinates;
    }

    public void setCoordinates(ArrayList<Float> coordinates) {
        this.coordinates = coordinates;
    }

}

File: app/src/main/java/com/example/atlasdevicesync/bean/Posizione.kt
package com.example.atlasdevicesync.bean

import io.realm.kotlin.ext.realmDictionaryOf
import io.realm.kotlin.ext.realmListOf
import io.realm.kotlin.internal.interop.RealmPointer
import io.realm.kotlin.types.EmbeddedRealmObject
import io.realm.kotlin.types.RealmDictionary
import io.realm.kotlin.types.RealmList

open class Posizione() : EmbeddedRealmObject {

    var type: String? = null
    var coordinates: Rea<Float> = realmDictionaryOf()
}